docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2218,6 +2234,74 @@ Rules with `$removeparam` modifier are supported by AdGuard for Windows, Mac and
 
 :::
 
+:::caution Restrictions
+
+[AdGuard Browser Extension for Chrome MV3](/general/adguard-browser-extension/mv3-version) has some limitations:
+
+- Regular expressions, negation and allowlist rules are not supported.
+- Group of similar `$removeparam` rules will be combined into one. Example:
+
+```bash
```

Comment:
The formatting seems to be incorrect here and below.

If you want the content to be on the same level as the list item, you need to pad it with 4 spaces.